vault/generate_root.go

```diff
@@ -145,6 +146,9 @@ func (c *Core) GenerateRootInit(otp, pgpKey string, strategy GenerateRootStrateg
 	if c.Sealed() && !c.recoveryMode {
 		return consts.ErrSealed
 	}
+	if !c.Sealed() && c.recoveryMode {
+		return errors.New("attempt to generate recovery token when already unsealed")
```

Comment:
Can we call it `recovery operation token`?

Reply:
I liked that idea when we talked about it previously, but if the command line option and the api path say `recovery-token` I'm not sure we gain clarity by calling it a "recovery operation token" in the logs/errors.

Reply:
I don't think we can rely on core being unsealed. We may have to do `!c.barrier.Sealed()` check here.

Reply:
Mind you I guess there's precedent with dr-token.  Ok, will fix.